.github/auto-comment-config.yaml

```diff
@@ -0,0 +1,7 @@
+label_mappings:
+  - labels: 
+      - hold
+    comment: Wait, please do not proceed this yet.
+  - labels: 
+      - lang/en
+    comment: To see where this is in the review pipeline and follow the progress, please have a look at the [project board](https://github.com/orgs/cncf/projects/26/views/1).
```

Comment:
```suggestion
    comment: To see where this is in the review pipeline and follow the progress, please look at the [definition review board](https://github.com/orgs/cncf/projects/26/views/1).
```

Reply:
Thanks @nate-double-u for the enhancement!
Applied your comment as suggested!
- https://github.com/cncf/glossary/pull/2389/files#diff-0d826e18f815cafca3a28b7926fc6430b2ba6d52ad4289656a5201e2561dafe1R4
- https://github.com/cncf/glossary/pull/2389/files#diff-0d826e18f815cafca3a28b7926fc6430b2ba6d52ad4289656a5201e2561dafe1R7